src/data/download_links.yml

```diff
@@ -1,64 +1,64 @@
-release_notes: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.7"
+release_notes: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.8"
 macos:
   cli:
     msg: "CLI client for macOS"
-    example_filename: "brclient-darwin-amd64-v0.1.7"
+    example_filename: "brclient-darwin-amd64-v0.1.8"
     links:
       -
         os: macOS
         arch: amd64
-        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.1.7/brclient-darwin-amd64-v0.1.7"
+        url: "https://github.com/companyzero/bisonrelay/releases/download/v0.1.8/brclient-darwin-amd64-0.1.8"
```

Comment:
It is better to fix the inconsistent filenames at their origin and add `v` to the version suffix: `-v0.1.8`.

Otherwise it breaks `example_filename` field above and makes `brclient` inconsistent with BR GUI, Decred and DEX releases.